todo-app/src/todo/entity/todo.entity.ts

```diff
@@ -0,0 +1,16 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity()
+export class Todo {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column({ type: 'varchar', nullable: false })
+  title: string;
+
+  @Column({ type: 'varchar', nullable: false })
+  userName: string;
+
+  @Column({ type: 'date', nullable: false })
+  todoDate: Date;
```

Comment:
- the prefix `todo` here is redundant 
- todo.todoDate seems pointless
- instead call it `dueDate` so it appears like `todo.dueDate`